src/datastore/database.hpp

```diff
@@ -0,0 +1,80 @@
+#pragma once
+
+#include <vector>
+#include "souvenir.hpp"
+#include "stadium.hpp"
+#include "team.hpp"
+
+/**
+ * @class Database class
+ *
+ * This class holds all of the programs data.
+ * It contains a map of teams and a map of stadiums.
+ * Each stadium has a vector of souvenirs within it's class.
+ *
+ * All the members aside from getEnumValue are static
+ */
+class Database
+{
+public:
+    /* Constructor */
+    Database();
+
+    /* Destructor */
+    ~Database();
```

Comment:
Destructor isn't needed for singletons since they're destroyed at he end of program execution

Reply:
k